### DISCUSSION TITLE
FP 1.12.2_06: Tag category manager topic - how to save?

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1899] [General]
**Describe the bug**

the fact that every time I'm not able to save correctly my tags it became for me a continuous open-close-reopen the tag category manager editor and check if everything is as expected, sometimes the enter save the tag sometimes not at now I do not believe to the flag to the confirm and so on because I try all of those combination but if I repeat them I cannot achieve the same result... what is wrong?

Is it possible to clarify the role of the 3 different command in the editor: 
![image](https://github.com/freeplane/freeplane/assets/62674157/9c2272dd-c467-45f4-a4dd-5a163b5ee745)
1) what about "enter confirms"
2) what about "OK"
3) what about "cancel"
4) how to use the enter key and what is the expected designed effect of it

I place this note here as a bug and not in the forum because I think that this sort of instability is due to the TCME instability.
Because I use a lot tags, I need a feedback about it.
At now I'm in a middle of some sensitive projects and it will be really difficult to change the actual approach with using tags...
Thanks for you understanding

PS: If you think this topic must be approached in the forum we'll move there.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] w
 - Version [e.g. 22] 10 and 11
 - Java Version FP 1.12.2_06

**Comment by dpolivaev** [comment]
If enter confirms is checked, enter should have the same effect as clicking OK.
ESCAPE has the same effect as clicking CANCEL.

With OK your changes are applied to the mind map, with CANCEL they are forgotten.

Undo and redo are related to the whole set of changes.

We can have a separate discussion if it is the right approach or you prefer something else.

**Comment by Illegit** [comment]
Ok, thanks, It will be a later step.
At now it is necessary to restore the std way you designed currently in FP 1.12.2_06: with enter confirms flagged nor OK button neither Enter are saving the changes. (at least on my pcs w10 and w11). Note that FP 1.12.2_04 is working.
